hoverxref/extension.py

```diff
@@ -139,6 +140,57 @@ def setup_sphinx_tabs(app, config):
             app.disconnect(listener_id)
 
 
+def setup_intersphinx(app, config):
```

Comment:
Should we call this `disconnect_intersphinx` or something a bit more explicit?

Reply:
I'm following the pattern I've been using already with `setup_` as prefix for each requirement.